src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/LinkDefendantToClaimViaAuthenticationTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.web.client.HttpClientErrorException;
+import uk.gov.hmcts.cmc.claimstore.BaseIntegrationTest;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.fixtures.SampleUser;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.fixtures.SampleUserDetails;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@TestPropertySource(
+    properties = {
+        "core_case_data.api.url=false",
+        "feature_toggles.ccd_async_enabled=false",
+        "feature_toggles.ccd_enabled=false",
+    }
+)
+public class LinkDefendantToClaimViaAuthenticationTest extends BaseIntegrationTest {
+
+    @Test
+    public void shouldReturn200HttpStatusAndUpdatedClaimWhenLinkIsSuccessfullySet() throws Exception {
+
+        String username = "defendant.email@htcms.net";
+        String password = "password";
+
+        given(userService.authenticateUser(username, password)).willReturn(SampleUser.builder()
+            .withAuthorisation(BEARER_TOKEN)
+            .withUserDetails(SampleUserDetails.builder()
+                .withUserId(DEFENDANT_ID)
+                .withMail(DEFENDANT_EMAIL)
+                .withRoles("citizen", "letter-" + SampleClaim.LETTER_HOLDER_ID)
+                .build())
+            .build()
+        );
+
+        Claim claim = claimStore.saveClaim(SampleClaimData.builder().build());
+
+        String urlTemplate = String.format("/testing-support/claims/%s/defendant/%s/%s",
```

Comment:
"/testing-support/claims/%s/defendant/%s/%s" can this be static variable as it is repeated through out the tests?